drivers/display/ssd16xx.c

```diff
@@ -1063,7 +1138,9 @@ static struct ssd16xx_quirks quirks_solomon_ssd1681 = {
 		.height = DT_PROP(n, height),				\
 		.width = DT_PROP(n, width),				\
 		.rotation = DT_PROP(n, rotation),			\
+		.scan_y_reverse = DT_PROP(n, scan_y_reverse),	\
 		.tssv = DT_PROP_OR(n, tssv, 0),				\
+		.gdo_flags = DT_PROP_OR(n, gdo_flags, 0),			\
```

Comment:
Please keep the `\` characters aligned here across all lines

Reply:
Updated.